keps/sig-node/0035-20190130-topology-manager.md

```diff
@@ -339,37 +357,63 @@ _Figure: Topology Manager components._
 ![topology-manager-instantiation](https://user-images.githubusercontent.com/379372/47447526-945a7580-d772-11e8-9761-5213d745e852.png)
 
 _Figure: Topology Manager instantiation and inclusion in pod admit lifecycle._
- 
+
 ### Feature Gate and Kubelet Flags
- 
+
 A new feature gate will be added to enable the Topology Manager feature. This feature gate will be enabled in Kubelet, and will be disabled by default in the Alpha release.  
 
  * Proposed Feature Gate:  
   `--feature-gate=TopologyManager=true`  
- 
+
  This will be also followed by a Kubelet Flag for the Topology Manager Policy, which is described above. The `none` policy will be the default policy.
- 
+
  * Proposed Policy Flag:  
  `--topology-manager-policy=none|best-effort|restricted|single-numa-node`  
- 
+
 ### Changes to Existing Components
 
 1. Kubelet consults Topology Manager for pod admission (discussed above.)
-1. Add two implementations of Topology Manager interface and a feature gate.
+2. Add two implementations of Topology Manager interface and a feature gate.
     1. As much Topology Manager functionality as possible is stubbed when the
-       feature gate is disabled.
-    1. Add a functional Topology Manager that queries hint providers in order
+     feature gate is disabled.
+    2. Add a functional Topology Manager that queries hint providers in order
        to compute a preferred socket mask for each container.
```

Comment:
I think word socket is inappropriate here. I remember it was a mess (socket/NUMA node).

Reply:
It seems clear to change to a NUMA node.